sites/website/src/docs/resources/cheat-sheet.md

```diff
@@ -0,0 +1,695 @@
+---
+id: cheat-sheet
+title: Cheat Sheet
+sidebar_label: Cheat Sheet
+custom_edit_url: https://github.com/microsoft/fast/edit/master/sites/website/src/docs/resources/cheat-sheet.md
+---
+
+# Cheat Sheet
+
+## Packages:
+
+### [@microsoft/fast-element](https://www.fast.design/docs/fast-element/getting-started)
+
+**A library that is a lightweight means to easily build performant, memory-efficient, standards-compliant Web Components.**
+
+* Provides a thin layer of opinions on top of [Web Components](https://www.fast.design/docs/resources/why-web-components), lifting the level of abstraction just enough to make it easier and FASTer to [build components]((#building-components)).   
```

Comment:
```suggestion
* Provides a thin layer of opinion on top of [Web Components](https://www.fast.design/docs/resources/why-web-components), lifting the level of abstraction just enough to make it easier and faster to [build components]((#building-components)).
```

Just a suggestion - I like the nod to "FAST" with 'faster', but I'm not sure we want to make that a convention. Open to others feedback on that though.

Reply:
@chrisdholt That makes sense, thank you.  I grabbed the verbiage from the docs: https://www.fast.design/docs/introduction.  I can make the updates here as well:

<img width="1004" alt="Screen Shot 2021-12-14 at 11 36 44 AM" src="https://user-images.githubusercontent.com/35910788/146071196-71a0340b-eda0-4983-aa5e-5a20ecb4df8d.png">